### ISSUE TITLE
question about loss function 

### ISSUE THREAD

**Issue by fmthoker** [nawidsayed/Cross-and-Learn#1]
Hi,
Thanks for uploading the code of your paper on Github. 
Can you tell me why have you calculated final loss like this
loss_sim = 2*torch.mean(sim_false*(1-self.weight_pos) -
sim_true*self.weight_pos, dim=0) / half  ( multiplying by 2 and also
dividing by half whose value is 2).
Also when you sample a pair of videos. do they have to form different
classes or can be from the same class?

Thanks
